Ginger/GingerCoreCommon/ReporterLib/WorkSpaceReporterBase.cs

```diff
@@ -36,47 +36,62 @@ public abstract class WorkSpaceReporterBase
 
         public void ToConsole(eLogLevel logLevel, string message, Boolean overwriteCurrentLine = false)
         {
-           
+            // Check if the console is hidden
+            if (!Console.IsOutputRedirected && Console.WindowHeight == 0)
```

Comment:
why we are not allowing all logs to log to the console, why not only to skip Git status logs?